Zend/zend_opcode.c

```diff
@@ -709,9 +709,9 @@ static void zend_check_finally_breakout(zend_op_array *op_array, uint32_t op_num
 	}
 }
 
-static uint32_t zend_get_brk_cont_target(const zend_op_array *op_array, const zend_op *opline) {
-	int nest_levels = opline->op2.num;
-	int array_offset = opline->op1.num;
+static uint32_t zend_get_brk_cont_target(const zend_op *opline) {
+	uint32_t nest_levels = opline->op2.num;
+	uint32_t array_offset = opline->op1.num;
```

Comment:
Technically should be fine, but we use `int` and even `zend_long` in other places (e.g. `zend_compile_break_continue()`, `CG(context).current_brk_cont`, etc.). If this needs to be adjusted, all of them should be.